### PR TITLE
NOISSUE: Improve logging in `cmd/<svc>/main.go`

### PR DESCRIPTION
**Pull request by arvindh123**
# What type of PR is this?
This is a feature because it adds the following functionality: 
- Improve logging in `cmd/<svc>/main.go`

## What does this do?
- Improve logging in `cmd/<svc>/main.go`

## Which issue(s) does this PR fix/relate to?

- Related Issue #
- Resolves #

### CODE REVIEW

File: .github/workflows/check-generated-files.yml [hunk 1]
```diff
@@ -51,12 +51,11 @@ jobs:
...
               - "things/postgres/clients.go"
               - "things/things.go"
-              - "auth/authz.go"
-              - "auth/domains_client.go"
+              - "pkg/authz.go"
+              - "pkg/authn.go"
               - "auth/domains.go"
               - "auth/keys.go"
               - "auth/service.go"
-              - "auth/token_client.go"
```
Comment (felixgateru) [inline .github/workflows/check-generated-files.yml:59]: No need to remove this one as it should be checked

**Review by felixgateru — CHANGES_REQUESTED** [review]

File: .github/workflows/check-generated-files.yml (hunk 1 above)
Comment (arvindh123) [reply .github/workflows/check-generated-files.yml:59]: But there is no file at this `auth/token_client.go`

**Review by felixgateru — APPROVED** [review]
LGTM

**Review by dborovcanin — CHANGES_REQUESTED** [review]
Please resolve conflicts.

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**